openmc/mesh.py

```diff
@@ -1137,6 +1137,70 @@ def from_hdf5(cls, group):
 
         return mesh
 
+    @classmethod
+    def from_domain(
+        cls,
+        domain,
+        dimension=[100, 100, 100],
+        mesh_id=None,
+        phi_grid=[0.0, 2*pi],
+        name=''
+    ):
+        """Create mesh from an existing openmc cell, region, universe or
+        geometry by making use of the objects bounding box property. phi_grid
+        is set to have a range of 0 to 2π by default.
```

Comment:
Per Python [docstring conventions](https://peps.python.org/pep-0257/#multi-line-docstrings), you should have a one-line summary followed by a blank line, followed by a more elaborate description if necessary

Reply:
Thanks, I've reduced it to a single line